pkg/controller/add_assignmetadata.go

```diff
@@ -0,0 +1,24 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	assignmetadata "github.com/open-policy-agent/gatekeeper/pkg/controller/assignmetadata"
+)
+
+func init() {
```

Comment:
We should add flag-gating somewhere in this PR.

Reply:
Added a small PR for the flag gate: #930 (one that can hopefully get in asap)

Reply:
flag gate is merged